security/pkg/pki/util/generate_cert.go

```diff
@@ -430,3 +435,14 @@ func encodePem(isCSR bool, csrOrCert []byte, priv any, pkcs8 bool) (
 	err = nil
 	return
 }
+
+func EncodeCsrPem(isCSR bool, csrOrCert []byte, pkcs8 bool) (
```

Comment:
This new function is not used, could be removed?

Reply:
Thanks, removed